src/components/atomic-swap/ProposalBalanceTable.js

```diff
@@ -15,12 +15,17 @@ import { t } from "ttag";
  * @param {DisplayBalance[]} balance
  */
 export function ProposalBalanceTable ({ partialTx, wallet, balance }) {
-    const sendingBalances = balance.filter(b => b.sending > 0);
     const receivingBalances = balance.filter(b => b.receiving > 0);
+    // If the wallet participates in the proposal with a token with zero balance, it is displayed as "sending"
+    const sendingBalances = balance.filter(b => {
+        return b.sending > 0 || b.sending === -0;
+    });
 
     const renderRows = () => {
         const renderOne = (amount, symbol) => {
-            if (!amount || !symbol) return '';
+            if (!amount && !symbol) {
```

Comment:
Javascript understands amount `0` as `false`, so we need to accept a _falsey_ amount that comes with a symbol.